src/ceph-volume/ceph_volume/tests/util/test_system.py

```diff
@@ -226,6 +226,13 @@ def apply(value='/bin/restorecon'):
     return apply
 
 
+# python2 has no FileNotFoundError
```

Comment:
I take it that this should've failed with tox, just that we didn't run it before right?

Reply:
correct